sigstore/_internal/rekor/client.py

```diff
@@ -112,7 +112,7 @@ def entries(self) -> RekorEntries:
         Returns a `RekorEntries` capable of accessing detailed information
         about individual log entries.
         """
-        return RekorEntries(urljoin(self.url, "entries/"), session=self.session)
+        return RekorEntries(urljoin(self.url, "entries"), session=self.session)
```

Comment:
Sorry, I might have been unclear -- I was saying that we could remove `urljoin` entirely and do this:

```suggestion
        return RekorEntries(f"{self.url}/entries", session=self.session)
```

(And similarly for all other `urljoin` uses below.)

Reply:
removed urljoin